pkg/dynamicaccess/act.go

```diff
@@ -0,0 +1,55 @@
+package dynamicaccess
+
+import (
+	"encoding/hex"
+
+	"github.com/ethersphere/bee/pkg/manifest"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+type Act interface {
+	Add(lookupKey []byte, encryptedAccessKey []byte) Act
+	Get(lookupKey []byte) []byte
+	Load(lookupKey []byte) manifest.Entry
+	Store(me manifest.Entry)
+}
+
+var _ Act = (*defaultAct)(nil)
```

Comment:
This shouldn't be called `defaultAct`.
Don't use "default", when writing mocks.
// Viktor

Reply:
in progress: Ferenc

Reply:
PR: https://github.com/Solar-Punk-Ltd/bee/pull/17

Reply:
resolved